src/main/java/net/masterthought/cucumber/json/deserializers/EmbeddingDeserializer.java

```diff
@@ -25,8 +25,12 @@
     public Embedding deserialize(JsonNode rootNode, Configuration configuration) {
         String data = rootNode.get("data").asText();
         String mimeType = findMimeType(rootNode);
+        String name = null;
+        if (rootNode.has("name")) {
```

Comment:
move `name` to static variable

Reply:
I updated the PR with two constructors for Embedding instead.